source/index.html.md

```diff
@@ -1295,8 +1295,7 @@ Create a new Organization
   "forwarder": false,
   "debtor": false,
   "shipper": false,
-  "active": false,
-  "updated": "2020-01-01T00:00:00.000Z"
+  "active": false
```

Comment:
Remove also form line 1246

Reply:
But this is used on the GET method line 1323 ?

Reply:
oh ok, so it is still in that one. will approve